site/in-dev/cli.md

```diff
@@ -5,35 +5,55 @@ title: "Nessie CLI"
 # Nessie CLI
 
 The Nessie CLI is an easy way to get started with Nessie. It supports multiple branch 
-and tag management capabilities. This is installed as a standalone uber jar from the 
-[Nessie download page](../downloads/index.md) or the
-[releases page on GitHub](https://github.com/projectnessie/nessie/releases/). 
+and tag management capabilities. 
 
 Nessie CLI is designed to be usable as an interactive REPL supporting auto-completion,
 highlighting where appropriate and has built-in help. Long outputs, like a commit log,
 are automatically paged like the Unix `less` command.
 
+![Nessie CLI](../img/cli-intro.png)
 
+## Installation
 
-![Nessie CLI](../img/cli-intro.png)
+Nessie CLI can be installed in two different ways:
 
-## Usage
+=== "Standalone jar"
+    A standalone uber jar can be downloaded from the [Nessie download page] or the [releases page on GitHub].
+    Run it as follows:
+
+    ```bash
+    java -jar nessie-cli-::NESSIE_VERSION::.jar
+    ```
+
+    !!! info
+        Running the standalone uber jar requires Java 11 or higher.
 
-Nessie CLI requires Java 11.
+=== "Docker image"
+    Nessie CLI has a Docker image hosted on [GitHub Container Registry]. Images are also mirrored to [Quay.io].
 
-```bash
-java -jar nessie-cli-<version>.jar
-```
+    Running the docker image can be done as follows:
+    
+    ```bash
+    docker run --rm -it ghcr.io/projectnessie/nessie-cli:::NESSIE_VERSION::
```

Comment:
nit: maybe use `latest` for "in-dev" docs?

Reply:
But isn't this page also going to serve for released versions? Is there a way to input `latest` for nightly and a version number elsewhere?

Reply:
Took a stab at it: #8939

Reply:
> nit: maybe use `latest` for "in-dev" docs?

`in-dev` serves the current "development" docs _and_ becomes the released version docs.